js/model3D/shared/utils.ts

```diff
@@ -17,11 +18,14 @@ const create_camera = (
 	if (camera_position[2] !== null) {
 		helperCamera.radius = camera_position[2];
 	}
-	// Disable panning. Adapted from: https://playground.babylonjs.com/#4U6TVQ#3
-	helperCamera.panningSensibility = 0;
-	helperCamera.attachControl(false, false, -1);
-	helperCamera.pinchToPanMaxDistance = 0;
-	helperCamera.wheelPrecision = 2500 / zoom_speed;
+	helperCamera.lowerRadiusLimit = 0.1;
+	const updateCameraSensibility = (): void => {
+		helperCamera.wheelPrecision = (150 * zoom_speed) / helperCamera.radius;
```

Comment:
I believe increasing the wheel precision actually decreases the zoom speed: https://forum.babylonjs.com/t/zoom-sensitivity/10338/3, which is why I had it in the denominator before. What's the reason for the change?

Reply:
When I test these changes using the `demo/model3D/run.py` demo, I'm finding it bit too sensitive for me

Reply:
Thank you for your review! I agree with you concerns and adjusted the zoom computation.

Zoom speed is now correctly in the denominator, and I've set the speed to 250, which seems to approximately reflect that of [sketchfab](https://sketchfab.com/3d-models/deadly-melodic-d60f5ca4c0bf4794a5dee82a040ce99f) on my machine.